storages/mongo-storage/src/store.rs

```diff
@@ -231,9 +232,25 @@ impl MongoStorage {
                     }
                     .map(|is_primary| ColumnUniqueOption { is_primary });
 
-                    let default = doc
+                    let default_comment_patterns =
+                        Regex::new(r"^(?:DEFAULT\s*(.*?))?(?:\s*COMMENT\s*'(.*?)')?$")
+                            .map_err(|_| MongoStorageError::Unreachable)
+                            .map_storage_err()?;
+
+                    let (default, comment) = doc
                         .get_str("description")
                         .ok()
+                        .and_then(|desc| {
+                            default_comment_patterns.captures(desc).map(|cap| {
+                                let default = cap.get(1).map(|m| m.as_str().to_string());
+                                let comment = cap.get(2).map(|m| m.as_str().to_string());
+
+                                (default, comment)
+                            })
+                        })
+                        .unwrap_or((None, None));
+
+                    let default = default
                         .map(parse_expr)
                         .map(|expr| expr.and_then(|expr| translate_expr(&expr)))
                         .transpose()?;
```

Comment:
How about using serde with new `ColumnDescription` struct?
In #1400 , I was using serde for `TableDescription` (There are two kinda description fields; table and column).
It would be also helpful to extend more features of column someday.

```rust
      let column_description = doc.get_str("description");
      let ColumnDescription { default, comment } = match column_description {
          Ok(desc) => {
              serde_json::from_str::<ColumnDescription>(desc).map_storage_err()?
          }
          Err(ValueAccessError::NotPresent) => ColumnDescription {
              default: None,
              comment: None,
          },
          Err(_) => {
              return Err(Error::StorageMsg(
                  MongoStorageError::InvalidGlueType.to_string(),
              ))
          }
      };
```
```rust
// storages/mongo-storage/src/description.rs
use gluesql_core::ast::Expr;
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
pub struct ColumnDescription {
    pub default: Option<Expr>,
    pub comment: Option<String>,
}
```
and then,

(⚠️ 🦀 `fold` need to return result.. )
```rust
// storages/mongo-storage/src/store_mut.rs
    let column_description = description::ColumnDescription {
        default: column_def.default.clone(),
        comment: column_def.comment.clone(),
    };

    let description = to_string(&column_description).map_storage_err()?;

    property.extend(doc! {
        "description": description,
    });
```